python/ray/tune/syncer.py

```diff
@@ -681,37 +686,61 @@ def __getstate__(self):
 class _DefaultSyncer(_BackgroundSyncer):
     """Default syncer between local and remote storage, using `pyarrow.fs.copy_files`"""
 
-    def __init__(
-        self, storage_filesystem: Optional["pyarrow.fs.FileSystem"] = None, **kwargs
-    ):
+    def _sync_up_command(
+        self, local_path: str, uri: str, exclude: Optional[List] = None
+    ) -> Tuple[Callable, Dict]:
+        return (
+            upload_to_uri,
+            dict(local_path=local_path, uri=uri, exclude=exclude),
+        )
+
+    def _sync_down_command(self, uri: str, local_path: str) -> Tuple[Callable, Dict]:
+        return (
+            download_from_uri,
+            dict(uri=uri, local_path=local_path),
+        )
+
+    def _delete_command(self, uri: str) -> Tuple[Callable, Dict]:
+        return delete_at_uri, dict(uri=uri)
+
+
+class _FilesystemSyncer(_BackgroundSyncer):
```

Comment:
Here's the new syncer that doesn't depend on `remote_storage`.

Reply:
Let's move this class to the storage.py file?